Learn-Information-Security-by-Building-a-Contact-Form/server040.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
```

Comment:
There's no need to include a link to the Helmet homepage. Learners can find that on their own if they need to. As for line 6, you could instruct learners to write that themselves. This might be the first time learners see the Express `use` method, so we should explain what that is and a bit more about Helmet:

// Though it's just one layer of security, Helmet makes it easy to set your application's HTTP headers to smart, restrictive defaults to protect against common attacks.
// Helmet is actually a bundle of "middleware", or functions that we can add in ourselves instead of relying solely on what's included with Express.
// To load a middleware function in an Express app, first call `app.use()`.

The next step could briefly explain what `dnsPrefetchControl` does and have learners pass `helmet.dnsPrefetchControl()` to `app.use()`.